From jorrit@ncode.nl  Tue Jun 23 14:48:03 2009
From: Jorrit Schippers <jorrit@ncode.nl>
Date: Wed, 10 Jun 2009 15:34:26 +0200
Subject: Staging: rt2870: Add USB ID for Sitecom WL-608
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org, Jorrit Schippers <jorrit@ncode.nl>
Message-ID: <1244640866-21385-1-git-send-email-jorrit@ncode.nl>


Add the USB id 0x0DF6,0x003F to the rt2870.h file such that the
Sitecom WL-608 device will be recognized by this driver.

Signed-off-by: Jorrit Schippers <jorrit@ncode.nl>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2870/rt2870.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rt2870/rt2870.h
+++ b/drivers/staging/rt2870/rt2870.h
@@ -89,6 +89,7 @@
 	{USB_DEVICE(0x0DF6,0x002C)}, /* Sitecom */		\
 	{USB_DEVICE(0x0DF6,0x002D)}, /* Sitecom */		\
 	{USB_DEVICE(0x0DF6,0x0039)}, /* Sitecom */		\
+	{USB_DEVICE(0x0DF6,0x003F)}, /* Sitecom WL-608 */	\
 	{USB_DEVICE(0x14B2,0x3C06)}, /* Conceptronic */		\
 	{USB_DEVICE(0x14B2,0x3C28)}, /* Conceptronic */		\
 	{USB_DEVICE(0x2019,0xED06)}, /* Planex Communications, Inc. */		\
